abs.yaml

```
@@ -0,0 +1,16 @@
+aggregate_check: false
+
+upload_channels: [r]
+
+pbp_only_deployment: prod
+
+build_parameters:
+  - "--R 4.3.1"
```

```suggestion
build_parameters:
  - "--suppress-variables"
  - "--skip-existing"
  - "--error-overlinking"
  - "--variant-config-files ./aggregateR/conda_build_config.yaml 
```

I think we should be able to get something working that way. By removing from aggregate cbc all the R related entries, and removing from aggregateR all the pinnings already in the aggregate cbc.

---

Sounds good. Let me try.

---

Nope.
`FileNotFoundError: [Errno 2] No such file or directory: '/feedstock/.abs/aggregateR/conda_build_config.yaml'`

---

Hmm, it's more likely to be `../conda_build_config.yaml`.
It's not entirely clear what's happening but reversing the logs:

```
2025-04-02T13:57:36.055184+00:00  | INFO     |   Running command: ['docker', 'run', '--init', ... '-v', '/tmp/abs_1eu91akdka/clone:/feedstock', ... '-w', '/feedstock/.abs', 'public.ecr.aws/y0o4y9o3/anaconda-pkg-build:latest', '/bin/bash', '-c', '\'conda build "/feedstock/r-ijtiff-feedstock" ... --suppress-variables --skip-existing --error-overlinking --variant-config-files ./aggregateR/conda_build_config.yaml ...\'']
```

PWD is `/feedstock/.abs` and `/feedstock` is `/tmp/abs_1eu91akdka/clone` where earlier we had

```
2025-04-02T13:57:28.782629+00:00  | INFO     |   Running command: git -C "/tmp/abs_1eu91akdka/clone" remote add origin git@github.com:AnacondaRecipes/aggregateR.git
```

and then some `pkg_build` thing was fetched... 🤷 

Anyway, `../conda_build_config.yaml` looks more plausible.

---

Let me verify your idea!

---

It's still not working without `- "--R 4.3.1"`. Adding it back with a comment

---

I'll try a bit around cbc settings before approving.

---

Okay... I disabled `"--R 4.3.1"` in `abs.yaml` and used `r-base {{ r_version }}` in `host`. It solved the issue.

---

I also toyed with your PR. The best is to add r-base to aggregateR's cbc config. That way future rebuilds will require less recipe update (only bump in build number). If you update aggregate's cbc libtiff pin, you also won't have to set 4.7 explicitly in the recipe.

---

I added r`-base` to `aggregateR`'s `cbc.yaml`

---

`Aggregate`'s cbc.yaml with `libtiff 4.7` is ready for review https://github.com/AnacondaRecipes/aggregate/pull/333